ast/ast.fbs

```diff
@@ -116,6 +122,7 @@ union Expression {
   UnaryExpression,
   UnsignedIntegerLiteral,
   BadExpression,
+  VectorExpression,
```

Comment:
You don't need to add a vector expression to the flatbuffer schema. This work should only be about creating the `vector` type.